test/redirects.test.js

```diff
@@ -404,4 +405,98 @@ lab.experiment('hapi-redirect', () => {
       });
     });
   });
+
+  lab.test(' accepts a callback that adds additional dynamic routes', (done) => {
+    let count = 0;
+    server.register({
+      register: redirectModule,
+      options: {
+        log: true,
+        log404: true,
+        redirects: {
+          '/test301': '/it/works',
+        },
+        getRedirects(slug, redirectDone) {

```

Comment:
What is slug?

Reply:
Think 'slug' this was left over from the pagedata redirects.   I'm passing the plugin options to this callback now.